### PR TITLE
 Take into account emails for domain extraction

### PR DESCRIPTION
**Pull request by meirwah**

### CODE REVIEW

File: Scripts/script-ExtractDomain.yml [hunk 1]
```diff
@@ -155,8 +199,14 @@ args:
 - name: tldList
   description: Name of Demisto list to hold known TLD list, so will be ignored when
     looking for domains
+- name: extractFromEmails
+  auto: PREDEFINED
+  predefined:
```
Comment (aviadl) [inline Scripts/script-ExtractDomain.yml]: Need to add default values `true` to markAsIndicator and `calcReputation` - then you will not need the logic at the top of the script

File: Scripts/script-ExtractDomain.yml [hunk 2]
```diff
@@ -13,10 +25,76 @@ script: |-
           reg = reg.substring(1);
           return str.match(reg);
       }
-
```
Comment (aviadl) [inline Scripts/script-ExtractDomain.yml]: For regex match is the logic with the `~` prefix still valid at 3.5?

File: Scripts/script-ExtractDomain.yml [hunk 3]
```diff
@@ -13,10 +25,76 @@ script: |-
...
       return false;
   };
 
+  var addToContextAndMD = function(domains,ec,md) {
```
Comment (aviadl) [inline Scripts/script-ExtractDomain.yml]: add spaces between parameters `domains, ec, md`

File: Scripts/script-ExtractDomain.yml (hunk 2 above)
Comment (meirwah) [reply Scripts/script-ExtractDomain.yml]: It works with backward too

**Comment by meirwah** [comment]
@aviadl plz review fixes

**Pull request merged**